presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentOptions.java

```diff
@@ -24,7 +24,7 @@
     @Option(names = "--config", paramLabel = "<config>", description = "Environment config to use")
     public String config = "config-default";
 
-    @Option(names = "--server-package", paramLabel = "<package>", description = "Path to Presto server package " + DEFAULT_VALUE, defaultValue = "presto-server/target/presto-server-${project.version}.tar.gz")
+    @Option(names = "--server-package", paramLabel = "<package>", description = "Path to Presto server package " + DEFAULT_VALUE, defaultValue = "${server.module}/target/${server.module}-${project.version}.tar.gz")
```

Comment:
I think we need a new `server.name` variable here in case the server module name is different than the produced tarball name.
```
${server.module}/target/${server.name}-${project.version}.tar.gz